typescript/src/start_script.ts

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { execSync } from 'child_process';
+import * as path from 'path';
+import { argv } from 'yargs';
+import * as fs from 'fs-extra';
+import { Dirent } from 'fs-extra';
+
+const GOOGLE_GAX_PROTOS_DIR = path.join(
+  __dirname,
+  '..',
+  '..',
+  'node_modules',
+  'google-gax',
+  'protos'
+);
+
+// Add folder of plugin to PATH
+process.env['PATH'] = __dirname + path.delimiter + process.env['PATH'];
+
+let outputDir = '';
+if (argv.output_dir) {
+  outputDir = argv.output_dir as string;
+} else {
+  process.exit(1);
```

Comment:
We are doing great for robots (`process.exit(1)`) but not too good for humans (no error message) :)  Let's add `console.error('Output directory is required: --output_dir path');` before exiting.

Reply:
Sounds good to me, it makes more sense to give error message before exit.